data_scripts/get_additional_wq.py

```diff
@@ -7,15 +7,30 @@
 from django.core.wsgi import get_wsgi_application
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "streamwebs_frontend.settings")
-proj_path = "/opt/streamwebs/streamwebs_frontend/"
+proj_path = "/home/centos/streamwebs/streamwebs_frontend/"
```

Comment:
I changed the dockerfile/docker-compose to set the project up in ``/home/centos`` to match Openstack. 

How do I set the path from config?